web/styles/app_components.css

```diff
@@ -823,8 +823,14 @@ input.settings_text_input {
         }
     }
 
-    tbody > tr:nth-child(odd) > td {
-        background-color: var(--color-background-modal);
+    tbody > tr:nth-child(odd) {
+        td {
+            background-color: var(--color-background-modal);
+        }
+
+        th {
+            background-color: var(--color-background-tbody-th-odd);
+        }
```

Comment:
I'm not able to test this(--color-background-tbody-th-odd) out in the app, can't find a table that fulfills this, some help would be appreciated!

Reply:
That would indicate a `<th>` element as part of a row in the body of the element. I don't see anywhere in the codebase where we do that (effectively, it would be for a heading row down the lefthand side of a table)...the instances I've seen of `<th>` are all part of `<thead>`. My guess is that this was a selector stack from Bootstrap that just got carried forward.